core/src/main/java/com/datastax/oss/driver/internal/core/cql/Conversions.java

```diff
@@ -142,6 +142,12 @@ public static Message toMessage(
     ProtocolVersion protocolVersion = context.getProtocolVersion();
     ProtocolVersionRegistry protocolVersionRegistry = context.getProtocolVersionRegistry();
     CqlIdentifier keyspace = statement.getKeyspace();
+    int nowInSeconds = statement.getNowInSeconds();
+    if (nowInSeconds != Integer.MIN_VALUE
```

Comment:
I would extract `Integer.MIN_VALUE` and name it as a constant, for example:
`NOW_IN_SECONDS_UNSET` because it is used in a lot of places